winpr/libwinpr/wtsapi/wtsapi.c

```diff
@@ -52,9 +52,15 @@ static PWtsApiFunctionTable g_WtsApi = NULL;
 static HMODULE g_WtsApi32Module = NULL;
 static WtsApiFunctionTable WtsApi32_WtsApiFunctionTable = { 0 };
 
+#if defined(_WIN32) && defined(__MINGW32__)
 #define WTSAPI32_LOAD_PROC(_name, _type)                                                   \
+	WtsApi32_WtsApiFunctionTable.p##_name = (_type)GetProcAddress(g_WtsApi32Module, "WT" \
```

Comment:
This looks strange, this is `C` macro expansion, why is this necessary?

Reply:
MingW GCC 9 preprocessor dislikes the original declaration variant, and considers it a syntax error...

Reply:
Interesting, this should not happen, might be a undesired preprocessor behaviour, need to check this.